Makefile

```diff
@@ -0,0 +1,39 @@
+########################################################
+
+NAME := ftl_installer
+
+# VERSION file provides one place to update the software version.
+VERSION := $(shell cat VERSION)
+
+__init__.py: ftl_installer/__init__.py.in
+	sed "s/%VERSION%/$(VERSION)/" $< > ftl_installer/__init__.py
+
+setup.py: setup.py.in
+	sed "s/%VERSION%/$(VERSION)/" $< > setup.py
+
+venv: clean __init__.py setup.py
+	@echo "#############################################"
+	@echo "# Creating a virtualenv"
+	@echo "#############################################"
+	virtualenv venv
```

Comment:
We probably want to allow the user to create the virtualenv with a different version of python (will help with testing and troubleshooting travis test failures).

`virtualenv -p python3 venv` or `virtualenv -p python2.7 venv` for example.